pages/index.js

```diff
@@ -47,29 +48,19 @@ export default function Home() {
                     />
                 </svg>
                 <div className="mx-auto justify-center grid sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-12">
-                    <Card
-                        name="Student Name"
-                        title="Lorem Ipsum Dddddddd"
-                        projectType="Website"
-                        thumbnail="https://source.unsplash.com/random"
-                        profileIcon="https://picsum.photos/200"
-                        award="gold"
-                    />
-                    <Card
-                        name="Student Name"
-                        title="Lorem Ipsum"
-                        projectType="Discord Bot"
-                        thumbnail="https://source.unsplash.com/random"
-                        profileIcon="https://picsum.photos/200"
-                        award="silver"
-                    />
-                    <Card
-                        name="Student Name"
-                        title="Lorem Ipsum"
-                        projectType="Website"
-                        thumbnail="https://source.unsplash.com/random"
-                        profileIcon="https://picsum.photos/200"
-                    />
+                    {RepoData.map(({ repoName, repoOwner, projectType, repoLogo, repoOwnerIcon, award }) => {
+                        return (
+                            <Card
+                                key={repoName}
+                                name={repoOwner}
+                                title={repoName}
+                                projectType={projectType}
+                                thumbnail={repoLogo ?? "https://avatars.githubusercontent.com/u/76849512?v=4"}
+                                profileIcon={repoOwnerIcon}
+                                award={award ?? "silver"}
+                            />
+                        );
```

Comment:
did the linter really put a ; here???

Reply:
Yeah? It's part of a return statement so ig i understand why it would